modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/search_container.js

```diff
@@ -119,6 +119,8 @@ AUI.add(
 
 							row.removeClass(CSS_TEMPLATE);
 
+							row.attr('id', instance.get('id') + '_' + id);
```

Comment:
Main concern here would be that the passed `id` is not unique and does not really generate a unique `id` in DOM. Based on usages that does not seem to be the case. If it were, `_instance._ids.push(id)` would likely be giving us trouble already somewhere down the road, so I'd say this should be good enough.